### PR TITLE
CORE-31590-2 default thirdPartyCookiesEnabled to true

### PR DESCRIPTION
**Pull request by wgamboa**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All tests pass and I've made any necessary test changes.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

**Comment by Aaronius** [comment]
Don't you need to update this code as well:

```
if (config.thirdPartyCookiesEnabled !== undefined) {
```

**Comment by wgamboa** [comment]
Thanks. That was careless of me.

File: src/components/Identity/index.js [hunk 1]
```diff
@@ -55,27 +55,31 @@ const createIdentity = ({ config, logger, cookieJar, network }) => {
...
             if (config.idSyncContainerId !== undefined) {
               identityQuery.identity.containerId = config.idSyncContainerId;
             }
+          }
 
-            if (config.thirdPartyCookiesEnabled !== undefined) {
-              identityQuery.identity.thirdPartyCookiesEnabled =
-                config.thirdPartyCookiesEnabled;
-            }
+          if (config.thirdPartyCookiesEnabled === false) {
```
Comment (Aaronius) [inline src/components/Identity/index.js]: I believe you should be able to safely do `if (!config.thirdPartyCookiesEnabled) {` here.

Comment (wgamboa) [reply src/components/Identity/index.js]: Thanks, will do.

**Pull request merged**